config/answers.sam-installer.yaml

```diff
@@ -0,0 +1,44 @@
+---
+certs:
+  generate: true
+  deploy: true
+  group: foreman
+katello: true
+foreman:
+  organizations_enabled: true
+  locations_enabled: true
+  initial_organization: Default Organization
+  initial_location: Default Location
+  custom_repo: true
+  configure_epel_repo: false
+  configure_scl_repo: false
+  ssl: true
+  server_ssl_cert: /etc/pki/katello/certs/katello-apache.crt
+  server_ssl_key: /etc/pki/katello/private/katello-apache.key
+  server_ssl_ca: /etc/pki/katello/certs/katello-default-ca.crt
+  server_ssl_chain: /etc/pki/katello/certs/katello-default-ca.crt
+  websockets_encrypt: true
+  websockets_ssl_key: /etc/pki/katello/private/katello-apache.key
+  websockets_ssl_cert: /etc/pki/katello/certs/katello-apache.crt
+  smart_proxy: false
+
+foreman::plugin::bootdisk: false
+foreman::plugin::discovery: false
+foreman::plugin::hooks: true
+foreman::plugin::tasks: true
+foreman::plugin::chef: false
+foreman::plugin::default_hostgroup: false
+foreman::plugin::puppetdb: false
+foreman::plugin::setup: false
+foreman::plugin::templates: false
```

Comment:
@ehelms will remove.....confirmed that default is false since plugin classes aren't referenced anywhere except here.